nextjs-app-v14/.cz-config.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
```

Comment:
I think I might have preferred renaming Next -> Next 13 and adding Next 14 to be separate PRs

Reply:
Is there any way we can share this file between the Next apps? Hopefully it should just be a case of importing and re-exporting